langchain/src/llms/googlevertexai.ts

```diff
@@ -64,6 +64,13 @@ export class GoogleVertexAI extends BaseLLM implements GoogleVertexAITextInput {
     super(fields ?? {});
 
     this.model = fields?.model ?? this.model;
+
+    // Change the defaults for code models
+    if (this.model.startsWith("code-")) {
+      this.temperature = 0.2;
```

Comment:
Is this recommended by Google?

Reply:
In conversations with Google, they said that 0.7 (which is what the configuration for the text model defaults to) is too high. They didn't have any specific suggestion for lower. My tests indicated that 0.5 produced poor results in my tests, but 0.2 produced good results.

Reply:
cool, thanks for the context